functions/ts/src/sops.ts

```diff
@@ -187,23 +248,49 @@ function readSopsArguments(configs: Configs) {
 
 sops.usage = `
 Sops function (see https://github.com/mozilla/sops).
-So far supports only decrypt operation:
-runs sops -d for all documents that have field 'sops:' and put the decrypted result back.
 
-Can be configured using a ConfigMap with the following flags:
-ignore-mac: true [Optional: default empty] Ignore Message Authentication Code during decryption.
+The fucntion supports 2 operations: encrypt and decrypt.
```

Comment:
Can you provide a typical function config example for each mode of operation? (Think of this as example in a MAN page so the user can get something working quickly without reading all of the docs).

Reply:
Done. I appreciate any feedback on how clear the examples I provided.